CONTRIBUTION

```diff
@@ -0,0 +1,26 @@
+Making Contributions
+
+Cothority is an Open Source program with many contributors (listed in the AUTHORS
+files), so we actively seek contributions (help on the Cothority users list,
+documentation, source code, ideas, …).  All contributions help us make a
+better product.
+
+All contributions without an explicit copyright statement or a CLAI/CLAC (see below)
+are assumed to be covered under a AGPL 2-Clause license as described in the
+file LICENSE.
+
+Developers who have contributed significant changes to the Cothority code must sign
+a Contributor License Agreement (CLAI), eventually also a CLAC if they work for
```

Comment:
Maybe it's already too late for that, but joinging on @Liamsi comment from our last HR meeting that this has been not really transparent, but... Don't you think that requiring one or even two signatures is too much, and on the contrary is more a "show stopper" than a incentive to contribute ? I'm just saying. But I'm truely unfamiliar with this whole patent/license world, but a lot of open source project don't require that, why do we do so ?